src/routers/legends/legends.service.ts

```diff
@@ -99,6 +99,7 @@ export class LegendsService {
 	}: GetLegendByNameDTO): Promise<
 		APIRes<LegendsEntity & { thumbnail: string }>
 	> {
+		legend_name.toLowerCase();

```

Comment:
Good idea but toLowerCase does not mutates the variable so you have re declare the variable like `const loverCaseLegenName = legend_name.toLowerCase();` or can use it directly in a function like `myFunc(arg1, arg2, legend_name.toLowerCase());`